### DISCUSSION TITLE
JupyterLab

### DISCUSSION THREAD

**Discussion by gabrielmagno** [catppuccin/catppuccin#2146] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

JupyterLab themes are provided as JupyterLab extensions, which are Python packages.

I started working on it, implementing first the Mocha flavor.

I'm almost finishing the Mocha port, then I plan to parametrize the colors to have all the 4 flavors available as 4 separated JupyterLab Themes.

My work in progress is in this repository: https://github.com/gabrielmagno/jupyterlab

Below is an screenshot of the current state of the port:
![image](https://github.com/catppuccin/catppuccin/assets/244890/f677c924-2750-4a4d-88f3-18224cd406a4)

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Closing as https://github.com/catppuccin/jupyterlab exists now! 🎉
